examples/sr_get_items_example.c

```diff
@@ -131,13 +132,18 @@ main(int argc, char **argv)
             ds = SR_DS_RUNNING;
         } else if (!strcmp(argv[2], "operational")) {
             ds = SR_DS_OPERATIONAL;
+        } else if (!strcmp(argv[2], "startup")) {
+            ds = SR_DS_STARTUP;
+        } else if (!strcmp(argv[2], "candidate")) {
+            ds = SR_DS_CANDIDATE;
         } else {
             printf("Invalid datastore %s\n", argv[2]);
             return EXIT_FAILURE;
         }
     }
+    op_str = argv[2] ? argv[2] : "running";
```

Comment:
I do not think this is safe, better use `argc > 2 ? argv[2] : "running"`.

Reply:
Done